models/iaf_psc_exp.cpp

```diff
@@ -372,15 +377,16 @@ nest::iaf_psc_exp::handle( SpikeEvent& e )
 {
   assert( e.get_delay_steps() > 0 );
 
+  const index input_buffer_slot = kernel().event_delivery_manager.get_modulo(
+    e.get_rel_delivery_steps( kernel().simulation_manager.get_slice_origin() ) );
+
   if ( e.get_weight() >= 0.0 )
```

Comment:
Here we are curiously inconsistent with *alpha: >= 0 vs > 0, without it making any difference in practice ;).

Reply:
It's consitsent now. Both neuron models now use the statement with the conditional operator suggested above.